src/linear_algebra/basic.lean

```diff
@@ -806,6 +806,16 @@ begin
   simp only [Sup_eq_supr', mem_supr_of_directed _ hdir.directed_coe, set_coe.exists, subtype.coe_mk]
 end
 
+/-- This is the statement that `coe : (submodule R M) → set M` is Scott continuous for the
+ω-complete partial order induced by the complete lattice structures. See
+`omega_complete_partial_order.continuous`. -/
```

Comment:
Is "scott continuous" worth a `def scott_continuous : Prop := sorry` somewhere?

Reply:
I think not, partly as there is already the `Prop` that I mention here `omega_complete_partial_order.continuous`. I'm not really qualified to comment though. Perhaps @cipher1024 has an opinion?

I will say that I only learned the term Scott continuous yesterday when I wrote this PR, and that I expect that most mathematicians will not know what an ω-complete partial order is (I didn't till yesterday).

Reply:
I don't know these terms either :)

Reply:
Can you add a version of the statement that uses [omega_complete_partial_order.continuous](https://leanprover-community.github.io/mathlib_docs/order/omega_complete_partial_order.html#omega_complete_partial_order.continuous)?

Reply:
OK I've done this with https://github.com/leanprover-community/mathlib/pull/7162/commits/796dd6e37d4ec7562d97b421cef2b7d5d4974fc9